dap-etl/transformation/src/test/resources/org/broadinstitute/monster/dap/common/transformation_helper/records/records.json

```diff
@@ -0,0 +1,18 @@
+{"record":"11111","redcap_event_name":"jan2020_arm_1","field_name":"study_id","value":"11111"}
```

Comment:
I am having trouble seeing where this is used 👀

Reply:
I commented below, but here is where it's [used](https://github.com/DataBiosphere/dog-aging-ingest/blob/fd86f096d386d971d6bdfcaee6fbea439de77006/dap-etl/transformation/src/test/scala/org/broadinstitute/monster/dap/common/TransformationHelperSpec.scala#L11)